.github/workflows/release.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           python-version: "3.x"
 
       - name: Install setuptools_scm
-        run: pip install 'setuptools_scm<7'
+        run: pip install "setuptools_scm<7" git-cliff
```

Comment:
why does setuptools_scm and git-cliff not live in the lock file?

Reply:
We do, but I'm afraid we don't have git cliff in the lock file. So we have to either install the complete environment and on top of that install git cliff, or just install these two.